src/scores/continuous/__init__.py

```diff
@@ -30,7 +36,7 @@
     "murphy_score",
     "murphy_thetas",
     "quantile_score",
-    "correlation",
+    "pearsonr",
```

Comment:
I think this might not be quite right. 'pearsonr' now sits in 'scores.continuous.correlation.pearsonr'. The 'correlation' in __all__ isn't causing any problem that I'm aware of, but presumably points to the correlation module whereas pre the last release was a score. I don't think the __all__ refer should point to pearsonr.

Reply:
Ah yes - I have reverted this.